cmd/jimmctl/cmd/migratemodel.go

```diff
@@ -20,15 +17,17 @@ import (
 
 const (
 	migrateModelCommandDoc = `
-The migrate command migrates a model(s) to a new controller. Specify
-a model-uuid to migrate and the destination controller name.
+The migrate commands migrates a model, or many models between two controllers
+registered within JIMM. 
+
+You may specify a model name (of the form owner/name) or model UUID.
```

Comment:
learning question: do we always use this format `owner/name`? Couldn't we use model name where there is no ambiguity? Otherwise it can be annoying to find the owner of a model every time.

Reply:
It's probably better to always prefix with name to guarantee it is distinct. We'd have to have a query that ensures only 1 of that model name exists otherwise to do this, so I think specifying owner is easier. Plus, people may very well often create models of the same name part of our model name... Unless we make model names completely distinct in and of themselves I don't think this is worth it.

Reply:
Just to chime in, it's a bug if you leave out the `owner` because Juju doesn't have unique model names, only `owner/model-name` is unique.